validator/must_present.go

```diff
@@ -0,0 +1,51 @@
+package validator
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/spf13/cobra"
+)
+
+type MustPresentRule struct {
+	Fields []string
+}
+
+func (p *MustPresentRule) Validate(cmd *cobra.Command) []error {
+
+	var errors []error
+	err := validateMustPresent(cmd, p)
+	errors = append(errors, err...)
```

Comment:
for visiting graph/tree best to pass info object and aggregate stats on it.
Number of commands checked. Errors.
Currently errors are string - we need this as struct with much more info